src/reducers/books.js

```diff
@@ -0,0 +1,12 @@
+const booksReducer = (state, action) => {
+  switch (action.type) {
+    case 'CREATE_BOOK':
+      return { books: [...state, action.payload] };
+    case 'REMOVE_BOOK':
+      return { books: [...state, action.payload] };
```

Comment:
This will add a new book instead of deleting it, change that to use the `filter` method filtering the state without the book you want to remove.